pyFAI/io/nexus.py

```diff
@@ -744,7 +744,7 @@ def save_NXcansas(filename, result,
             errors_ds.attrs["doc"] = "standard error of the mean"
             errors_ds.attrs["interpretation"] = "spectrum"
             errors_ds.attrs["units"] = "arbitrary"
-            # integration_data["errors"] = errors_ds
+            integration_data["errors"] = errors_ds
```

Comment:
Small nit: use a softlink instead of copying the data

Reply:
this does not duplicate (copy) the data, but makes a hard-link to the data.
https://docs.h5py.org/en/stable/high/group.html#hard-links

Reply:
Maybe I'll go for "I_errors" since I dislike the name "errors"